lib/context.js

```diff
@@ -278,6 +285,30 @@
             });
         },
 
+        /**
+         * Disable SSL certificate verification during REST API calls.
```

Comment:
Maybe reword so it doesn't just reflect `Disable`. I understand that the default flag is set to `true` and so that the method would only be used to disable the cert verification, though the naming of the method is a setter. 

e.g. "Whether or not to perform an SSL certification during REST API calls. @param defaults to `true`"